inc/alert.class.php

```diff
@@ -528,7 +531,7 @@ static function preItemForm($params = []) {
          $itemtype    = get_class($item);
          $entities_id = isset($params['item']->fields['entities_id'])
             ? $params['item']->fields['entities_id']
-            : 0;
+            : false; // false to use current entity
```

Comment:
OK for this. I pushed it in develop branch (8197caf72eb45f8023b225c5977df86dee33dcad).